packages/@react-stately/collections/src/CollectionBuilder.ts

```diff
@@ -109,7 +109,7 @@ export class CollectionBuilder<T extends object> {
     // Call this function to get a partial node, and recursively build a full node from there.
     if (React.isValidElement(element)) {
       let type = element.type as any;
-      if (typeof type !== 'function' || typeof type.getCollectionNode !== 'function') {
+      if (typeof type !== 'function' && typeof type.getCollectionNode !== 'function') {
```

Comment:
what caused this to surface as a bug?

Reply:
This was surfaced by Card.tsx where we needed to do a forwardRef and tack on a `.getCollectionNode` to support standalone cards and Cards within a CardView

Reply:
I assume ActionButton and others like this don't have the same issue because they aren't true collections?

Reply:
Well, ActionGroup doesn't use ActionButtons as children, it accepts `Item`s and then we render the ActionButton for them. `Item` doesn't use forwardRef because it doesn't accept refs, hence why this issue doesn't surface